website/about/index.html

```diff
@@ -0,0 +1,97 @@
+            open standards - this focus is a rejection of the world we live in of closed platforms and services that can\'t communicate with each other. We want our users to have freedom and choice in how
+            they communicate.') }}</p>
+        </section>
+      </div>
+    </div>
+  </section><!-- end #what -->
+
+  <section id="who">
+    <h2>{{ _('Who We Are') }}</h2>
+    <div class="row">
+      <div class="col full">
+        <p>{{ _('We are a community making communication free and open.') }}</p>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col full">
+        <section>
+          <h3>{{ _('Our Community') }}</h3>
+          <p>{{ _('There is a passionate team of volunteers involved in developing Thunderbird and assisting its users who would welcome your involvement. 
+          Check out our <a href="%(volunteer)s">Get Involved page</a> for how you can participate on the Thunderbird team. We also have paid staff working full-time on Thunderbird, 
+          <a href="https://donate.mozilla.org/thunderbird/">funded by donations</a> from our users. If you\'re curious about who is working on Thunderbird, check out 
+          the <a href="https://wiki.mozilla.org/Thunderbird/Core_Team">list of our core contributors</a>.')|format(volunteer=url('thunderbird.get-involved')) }}</p>
+        </section>
+        <section>
+          <h3>{{ _('Thunderbird Council') }}</h3>
+          <p>{{ _('The Thunderbird project is governed by seven councilors, elected by the Thunderbird community\'s active contributors. You can see the current elected Thunderbird Council members 
+          <a href="https://wiki.mozilla.org/Modules/Thunderbird#Thunderbird_Council">on the Thunderbird wiki</a>.') }}</p>
+        </section>
+      </div>
+    </div>
+  </section><!-- end #who -->
+
+  <section id="mozilla">
+    <h2>{{ _('Thunderbird and Mozilla Foundation') }}</h2>
+    <div class="row">
+      <div class="col full">
+        <p>{{ _('Thunderbird lives at the nonprofit <a href="https://foundation.mozilla.org/about/">Mozilla Foundation</a>.') }}</p>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col full">
+        <section>
+          <p>{{ _('Thunderbird is an independent project with its legal and financial home at <a href="https://foundation.mozilla.org/about/">the Mozilla Foundation</a>. The Mozilla Foundation believes the
+          Internet is a global public resource that must remain open and accessible to all. The Foundation is also the sole shareholder in the Mozilla Corporation, the maker of Firefox - but has its focus
+          on advocacy issues and keeping the Internet healthy. The Mozilla Foundation exerts no governance over the project, which instead falls to the Thunderbird Council, elected by the Thunderbird
+          project\'s community. The Mozilla Foundation became Thunderbird\'s home after an exploration of legal homes in 2017.') }}</p>
```

Comment:
I think you should link [exploration of legal homes in 2017](https://blog.mozilla.org/thunderbird/2017/05/thunderbirds-future-home/) to the appropriate blog article. Also, more unicode single quotes :)

Reply:
It is linked and unicode single quotes changed throughout document.